docs/conf.py

```diff
@@ -187,7 +187,7 @@
 # html_domain_indices = True
 
 # If false, no index is generated.
-# html_use_index = True
+html_use_index = False
```

Comment:
I don’t understand how this relates to the other changes?

Reply:
If I recall, this determines whether Sphinx generates an index of its own ([docs here](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_use_index)). If we're using Docsearch then we don't need to do that - I suppose it doesn't hurt to leave this on, although removing it might incrementally speed up the build.

Reply:
Thank you. I think this would be worth an additional comment.

Reply:
I have added a comment in c25f8afb1ec8a0f9c9aa0a6a8f963378ae808cd2.

Let me know if that is expressive enough.